index.src.html

```diff
@@ -2173,7 +2173,7 @@
 
 dictionary AudioDataInit {
   required AudioSampleFormat format;
-  [EnforceRange] required float sampleRate;
+  [EnforceRange] required unsigned long sampleRate;
```

Comment:
Sample-rate are float in the Web Audio API and, generally, in audio software. This saves lots of casts.

Reply:
> Additionally, if `sampleRate` is a float, should `{{AudioData/duration}}` be a float too?

No, this is specifically specified as integer microseconds.